detox/test/e2e/init.js

```diff
@@ -2,13 +2,18 @@ const detox = require('detox');
 const config = require('../package.json').detox;
 const adapter = require('detox/runners/jest/adapter');
 const specReporter = require('detox/runners/jest/specReporter');
+const assignReporter = require('detox/runners/jest/assignReporter');
 
 jasmine.getEnv().addReporter(adapter);
 
 // This takes care of generating status logs on a per-spec basis. By default, jest only reports at file-level.
 // This is strictly optional.
 jasmine.getEnv().addReporter(specReporter);
 
+// This will post which device has assigned to run a suite, in a multiple-worker test session.
+// This is strictly optional.
+jasmine.getEnv().addReporter(assignReporter);
```

Comment:
Is it justified to add and add and add new and new and new microreporters? What do you think?

Reply:
Should we unify to a 'detox' reporter for simplicity?